lib/unison-prelude/package.yaml

```diff
@@ -36,15 +37,28 @@ default-extensions:
   - BangPatterns
   - BlockArguments
   - ConstraintKinds
+  - DeriveAnyClass
```

Comment:
I updated extensions to be in sync with parser-typechecker so it's not confusing to move between packages.

Unless someone had a reason for using a smaller set?

Reply:
No reason; I've also found random inconsistencies all over. If the extension seems reasonable I just add it to the defaults like you did here. We have #4568 open, just waiting for someone to come through and standardize all of the lists. (It'd also be nice to keep that in sync somehow).